rpc/src/tests/ws.rs

```diff
@@ -136,7 +136,7 @@ mod testing {
 	}
 
 	#[test]
-	fn should_allow_initial_connection_but_only_once() {
+	fn should_not_allow_initial_connection_even_once() {
```

Comment:
I like the reference to the old test, but 'should_not_allow_initial_connection' would be simplier. You can ignore this grumble.